internal/cli/atlas/serverless/backup/restores/describe_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build unit
+
+package restores
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
+	"github.com/mongodb/mongodb-atlas-cli/internal/test"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestDescribeOpts_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockServerlessRestoreJobsDescriber(ctrl)
+
+	expected := &mongodbatlas.CloudProviderSnapshotRestoreJob{}
+
+	describeOpts := &DescribeOpts{
+		store:       mockStore,
+		clusterName: "Cluster0",
+		id:          "1",
+	}
+
+	mockStore.
+		EXPECT().
+		ServerlessRestoreJob(describeOpts.ProjectID, describeOpts.clusterName, describeOpts.id).
+		Return(expected, nil).
+		Times(1)
+
+	if err := describeOpts.Run(); err != nil {
+		t.Fatalf("Run() unexpected error: %v", err)
+	}
```

Comment:
Could you also check the template simial to this test? https://github.com/mongodb/mongodb-atlas-cli/blob/523f9a3363c2bb5554f2ba06d697b1ba710d0c87/internal/cli/alerts/describe_test.go#L106?